scripts/docs/fixtures/valid-aggregate/package.json

```
@@ -0,0 +1,5 @@
+{
+  "name": "docs-verifier-valid-fixture",
+  "version": "2.4.14",
```

ESLint

[error] Expected object keys to be in specified order. 'version' should be after 'type'.

(jsonc/sort-keys)